src/test/java/com/epam/tc/hw2/TestService.java

```diff
@@ -0,0 +1,96 @@
+package com.epam.tc.hw2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.util.Arrays;
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public class TestService {
+
+    protected WebDriverWait webDriverWait;
+    protected WebDriver driver;
+
+    @BeforeMethod
+    public void setUp() {
+
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        webDriverWait = new WebDriverWait(driver, 5);
+    }
+
+    //12. Close Browser
+    @AfterMethod
+    public void closeBrowser() {
+
+        webDriverWait = null;
+        driver.quit();
+    }
+
+    protected static final By SIGN_IN_BUTTON = By.xpath("//div[@class='profile-photo']/following-sibling::span");
+    protected static final By ENTER_BUTTON = By.xpath("//span[text()='Enter']");
+    protected static final By LOGIN_INPUT_BUTTON = By.id("name");
+    protected static final By PASSWORD_INPUT_BUTTON = By.id("password");
+    protected static final By USER_NAME = By.id("user-name");
+    protected static final By HOME_BUTTON = By.xpath("//a[text()='Home']");
+    protected static final By CONTACT_FORM_BUTTON = By.xpath("//a[text()='Contact form']");
+    protected static final By SERVICE_BUTTON = By.className("dropdown-toggle");
+    protected static final By METALS_AND_COLORS_BUTTON = By.cssSelector("a[href='metals-colors.html']");
```

Comment:
I suppose that all of this locators, could be merged into one

Reply:
These locators were merged into one.